From ca5794a4c375794c97cf3c185dda27529e5b3961 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Wed, 20 Jun 2018 14:12:42 -0400
Subject: [PATCH] Fix a makeguids building problem with generics.h.

Guard generics.h with EFIVAR_BUILD_ENVIRONMENT to keep it from
interfering with the makeguids build if libefivar.so isn't around
already.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/generics.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/generics.h b/src/generics.h
index e6af2fa..66be4bd 100644
--- a/src/generics.h
+++ b/src/generics.h
@@ -17,6 +17,7 @@
  *
  */
 
+#ifndef EFIVAR_BUILD_ENVIRONMENT
 #ifndef LIBEFIVAR_GENERIC_NEXT_VARIABLE_NAME_H
 #define LIBEFIVAR_GENERIC_NEXT_VARIABLE_NAME_H 1
 
@@ -182,3 +183,4 @@ generic_append_variable(efi_guid_t guid, const char *name,
 }
 
 #endif /* LIBEFIVAR_GENERIC_NEXT_VARIABLE_NAME_H */
+#endif /* EFIVAR_BUILD_ENVIRONMENT */
-- 
2.19.1

